ext/js/dictionary/dictionary-data-util.js

```diff
@@ -82,18 +82,57 @@ export function groupTermFrequencies(dictionaryEntry) {
     }
 
     const results = [];
+
+    /** @type {import('dictionary').AverageFrequencyListTerm} */
+    const averages = {};
     for (const [dictionary, map2] of map1.entries()) {
         const frequencies = [];
         const dictionaryAlias = aliasMap.get(dictionary) ?? dictionary;
-        for (const {term, reading, values} of map2.values()) {
+        for (let {term, reading, values} of map2.values()) {
```

Comment:
It would be better to keep this const and define new variables when you need to modify stuff.